Jenkinsfile

```diff
@@ -477,6 +477,17 @@ lock(resource: "build-${params.STREAM}") {
             }
         }
 
+        if (official) {
```

Comment:
Meh, let's just do it for all builds. That way if it breaks, we don't find out when we're trying to release.